setup/zero_position_calibration.py

```diff
@@ -0,0 +1,343 @@
+        10), interp_time=0.9, kp=180, kd=2.0)
+    time.sleep(0.5)
+
+    d_j1 = bump_to_limit(openarm, arm,  JointID.J1,
+                         step_deg=-0.2, kp=180, kd=2.1)
+    time.sleep(0.5)
+    interpolate(openarm, arm, JointID.J1, np.deg2rad(80))
+    time.sleep(0.5)
+
+    interpolate(openarm, arm, JointID.J4, -np.pi/2.0)
+    time.sleep(2.5)
+
+    ideal = {
+        JointID.J7: np.pi/2.0,
+        JointID.J6: np.pi/4.0,
+        JointID.J5: np.pi/2.0,
+        JointID.J4: 0.0,
+        JointID.J3: np.pi/2.0,
+        JointID.J2: np.deg2rad(-10),
+        JointID.J1: np.deg2rad(-80),
+    }
+    deltas = [d_j1, d_j2, d_j3, d_j4, d_j5, d_j6, d_j7, d_grip]
+    return ideal, deltas
+
+
+def _run_left_sequence(openarm, arm, grip):
+    d_grip = bump_to_limit(openarm, grip, JointID.GRIPPER)
+    time.sleep(0.5)
+    d_j4 = bump_to_limit(openarm, arm,  JointID.J4, step_deg=-0.2)
+    time.sleep(0.5)
+
+    interpolate(openarm, arm, JointID.J2, -np.deg2rad(5), interp_time=0.4)
+    time.sleep(0.5)
+    d_j3 = bump_to_limit(openarm, arm,  JointID.J3)
+    time.sleep(0.5)
+
+    interpolate(openarm, arm, JointID.J3, -
+                mech_lim[JointID.J3][1], interp_time=1.0)
+    time.sleep(0.5)
+    interpolate(openarm, arm, JointID.J2,  np.deg2rad(5),  interp_time=0.4)
+    time.sleep(0.5)
+    interpolate(openarm, arm, JointID.J4,  np.pi/2.0)
+    time.sleep(0.5)
+
+    d_j5 = bump_to_limit(openarm, arm,  JointID.J5)
+    time.sleep(0.5)
+    interpolate(openarm, arm, JointID.J5, -mech_lim[JointID.J5][1])
+    time.sleep(0.5)
+
+    d_j6 = bump_to_limit(openarm, arm,  JointID.J6)
+    time.sleep(0.5)
+    interpolate(openarm, arm, JointID.J6, -mech_lim[JointID.J6][1])
+    time.sleep(0.5)
+
+    d_j7 = bump_to_limit(openarm, arm,  JointID.J7)
+    time.sleep(0.5)
+    interpolate(openarm, arm, JointID.J7, -mech_lim[JointID.J7][1])
+    time.sleep(0.5)
+
+    d_j2 = bump_to_limit(openarm, arm,  JointID.J2)
+    time.sleep(0.5)
+    interpolate(openarm, arm, JointID.J2, -np.deg2rad(10),
+                interp_time=0.9, kp=180, kd=2.0)
+    time.sleep(0.5)
+
+    d_j1 = bump_to_limit(openarm, arm,  JointID.J1, kp=180, kd=2.1)
+    time.sleep(0.5)
+    interpolate(openarm, arm, JointID.J1, -np.deg2rad(80))
+    time.sleep(0.5)
+
+    interpolate(openarm, arm, JointID.J4, -np.pi/2.0)
+    time.sleep(2.5)
+
+    ideal = {
+        JointID.J7: np.pi/2.0,
+        JointID.J6: np.pi/4.0,
+        JointID.J5: np.pi/2.0,
+        JointID.J4: 0.0,
+        JointID.J3: np.pi/2.0,
+        JointID.J2: np.deg2rad(10),
+        JointID.J1: np.deg2rad(80),
+    }
+    deltas = [d_j1, d_j2, d_j3, d_j4, d_j5, d_j6, d_j7, d_grip]
+    return ideal, deltas
+
+# ---------- Main ----------
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Zero-pos calibration (compact)')
+    parser.add_argument('--canport', type=str, default='can0')
+    parser.add_argument('--arm_side', type=str,
```

Comment:
How about using `--XXX-YYY` style?

```suggestion
    parser.add_argument('--arm-side', type=str,
```